docs/marionette.region.md

```diff
@@ -74,183 +74,87 @@ var MyView = Marionette.View.extend({
+To overwrite the parent `el` of the region with the rendered contents of the
+inner View, use `replaceElement` as so:
 
-var MyRegion = Marionette.Region.extend({
-  el: '#main-nav'  // This is mandatory
-});
+```javascript
+var Mn = require('backbone.marionette');
 
-var MyView = Marionette.View.extend({
-  regions: {
-    mainRegion: MyRegion
-  }
+var OverWriteView = Mn.View.extend({
+  className: '.new-class'
 });
-```
-
-### Object Literal
-
-Finally, you can define regions with an object literal. Object
-literal definitions normally expect a `selector` or `el`
-property.  The `selector` property is a selector string, and
-the `el` property can be a selector string, a jQuery object,
-or an HTML node.
-
-You may also supply a `regionClass` property for a custom region
-class. If your `regionClass` already has `el` set, then you do
-not need to supply a `selector` or `el` property on the object
-literal.
 
-Any other properties you set on the object literal will be
-used as options passed to the region instance, including the
-`allowMissingEl` option.
-
-Ordinarily regions enforce the presence of a backing DOM element.
-In some instances it may be desirable to allow regions to be
-instantiated and used without an element, such as when regions
-defined by a parent `LayoutView` class are used by only some of its
-subclasses. In these instances, the region can be defined with the
-`allowMissingEl` option, suppressing the missing element error and
-causing `show` calls to the region to be treated as no-ops.
-
-```js
-var Marionette = require('backbone.marionette');
-
-var MyRegion = Marionette.Region.extend();
-var MyOtherRegion = Marionette.Region.extend();
-var MyElRegion = Marionette.Region.extend({ el: '#footer' });
-
-
-var MyView = Marionette.View.extend({
+var MyView = Mn.View.extend({
   regions: {
-    contentRegion: {
-      el: '#content',
-      regionClass: MyRegion
-    },
-
-    navigationRegion: {
-      el: '#navigation',
-      regionClass: MyOtherRegion,
-
-      // Options passed to instance of `MyOtherRegion` for
-      // the `navigationRegion` on `App`
-      navigationOption: 42,
-      anotherNavigationOption: 'foo'
-    },
-
-    footerRegion: {
-      regionClass: MyElRegion
+    main: {
+      el: '.overwrite-me',
+      replaceElement: true
     }
-  }
-});
-```
-
-Take note that one of the primary benefits of using `regionClass`
-with an `el` already set is to also provide options to the region
-instance. This isn't possible when using the region class directly
-like earlier.
-
-```js
-var Marionette = require('backbone.marionette');
-
-var MyRegion = Marionette.Region.extend({
-  el: '#content',
-});
+  },
 
-var MyView = Marionette.View.extend({
-  regions: {
-    contentRegion: {
-      regionClass: MyRegion,
-      myRegionOption: 'bar',
-      myOtherRegionOption: 'baz'
-    }    
+  onRender: function() {
+    this.showChildView('tooverwrite', new OverWriteView());
```

Comment:
😕 Shouldn't this be `this.showChildView('main', new OverWriteView());`
Or should the `main` key be renamed `tooverwrite` in `regions`?